src/reference/90-Developers-Guide/02-Coding-Guideline/00.md

```diff
@@ -90,83 +90,87 @@ Use [MiMa][mima].
 
 #### Public traits should contain `def` declarations only
 
+TODO: "results to codegen at subclass" is unclear
+
 - `val` or `var` in `trait` results to codegen at subclass and at the artificial `Foo\$class.\$init\$`.
 - `lazy val` results to codegen at subclass
 
 #### Abstract classes are also useful
 
 [To trait, or not to trait?][pins_trait127].
-It is less flexible compared to trait, but easier to maintain binary compatibility. It also has better Java interop.
+Abtract classes are less flexible compared to traits, but ease maintaining binary compatibility. They also have better Java interoperability.
 
-#### Seal the traits and abstract classes
+#### Seal traits and abstract classes
 
-If there's no need to keep the classes open, seal it.
+If there's no need to keep a class open, seal it.
 
 #### Finalize the leaf classes
 
-If there's no need to keep the classes open, finalize it.
+If there's no need to keep a class open, finalize it.
 
 #### Typeclass and subclass inheritance
 
-Typeclass pattern with pure trait might be easier to maintain binary compatibility than subclassing.
+The typeclass pattern with pure traits might be ease maintaining binary compatibility more so than subclassing.
 
 #### Avoid case classes, use sbt-datatype
 
-case class involves various codegen, which makes it harder to maintain binary compatibility over time.
+`case class` involves code generation that makes it harder to maintain binary compatibility over time.
```

Comment:
Just my opinion: in formal public documentation it's best to expand abbreviations, at least on first use, and try to avoid jargon.

Keep in mind also that people reading this might not *yet* be sbt developers (e.g. me :smiley:).